modules/test/playwright/tests/osb-faro-web/search-terms.spec.ts

```diff
@@ -0,0 +1,148 @@
+import {loginTest} from '../../fixtures/loginTest';
+import getRandomString from '../../utils/getRandomString';
+import {syncAnalyticsCloud} from '../analytics-settings-web/utils/analyticsSettings';
+import {navigateToACSitesPageViaURL} from './utils/navigation';
+import {CardSelectors} from './utils/selectors';
+import {changeTimeFilter} from './utils/time-filter';
+import {waitForLoading} from './utils/loading';
+
+import {liferayConfig} from '../../liferay.config';
+import {createIndividuals} from './utils/individuals';
+import {viewNameOnTableList} from './utils/utils';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	dataApiHelpersTest,
+	featureFlagsTest({
+		'LPS-178052': true,
+	}),
+	loginAnalyticsCloudTest(),
+	loginTest()
+);
+
+test('Verify that clicking the "All Search Terms" link redirects the user to the Search Terms tab with the same time filter previously set.', async ({
+	apiHelpers,
+	page,
+}) => {
+	const channelName = 'My Property - ' + getRandomString();
+
+	const {channel, project} = await syncAnalyticsCloud({
+		apiHelpers,
+		channelName,
+		page,
+	});
+
+	const individualName = 'ac';
+
+	const date = new Date();
+
+	const generateIndividual = (name) => {
+		const id = getRandomString();
+
+		return {
+			id,
+			name,
+		};
+	};
+
+	const individuals = [generateIndividual(individualName)];
+
+	await test.step('Create Individuals', async () => {
+		await createIndividuals({
+			apiHelpers,
+			individuals,
+		});
+	});
+
+	await test.step('Create Individuals Events', async () => {
+		const individualsEvent = individuals.map((individual) => ({
+			applicationId: 'Page',
+			canonicalUrl: `https://${liferayConfig.environment.baseUrl}/search`,
+			channelId: channel.id,
+			eventDate: date.toISOString(),
+			eventId: 'pageViewed',
+			title: 'Liferay',
+			url: `https://${liferayConfig.environment.baseUrl}/search?q=test`,
+			userId: individual.id,
+		}));
+
+		await apiHelpers.jsonWebServicesOSBAsah.createEvents(individualsEvent);
+	});
+
+	await test.step('Create Individuals Session', async () => {
+		const sessionIndividuals = individuals.map((individual) => ({
+			channelId: channel.id,
+			id: individual.id,
+			sessionEnd: date.toISOString(),
+			sessionStart: date.toISOString(),
+			userId: individual.id,
+		}));
+
+		await apiHelpers.jsonWebServicesOSBAsah.createSessions(
+			sessionIndividuals
+		);
+	});
+
+	await test.step('Go to Analytics Cloud and Switch the property', async () => {
+		await navigateToACSitesPageViaURL({
+			channelID: channel.id,
+			page,
+			projectID: project.groupId,
+		});
+	});
+
+	await test.step('Change time filter in Search Terms to Last 24 Hours and click on "All Search Terms" link', async () => {
+		await changeTimeFilter({
+			cardSelector: CardSelectors.SearchTerms,
+			page,
+			timeFilterPeriod: 'Last 24 hours',
+		});
+
+		await page.getByRole('link', {name: 'All Search Terms'}).click();
+
+		await waitForLoading(page);
```

Comment:
Hey @fmarcio, I think we already have a function to do this

```suggestion
		await navigateTo({
			page,
			pageName: 'All Search Terms',
		});
```

Reply:
done